file-formats/sajt/type/zif_aff_sajt_v1.intf.abap

```diff
@@ -0,0 +1,103 @@
+INTERFACE zif_aff_sajt_v1
+  PUBLIC.
+
+  TYPES:
+    "! <p class="shorttext">Header for SAJT Object</p>
+    "! The header for an application job template
+    BEGIN OF ty_header,
+      "! <p class="shorttext">Description</p>
+      "! Description of the application job template
+      "! $required
+      description           TYPE c LENGTH 120,
+      "! <p class="shorttext">Original Language</p>
+      "! Original language of the application job template
+      "! $required
+      original_language     TYPE sy-langu,
+      "! <p class="shorttext">ABAP Language Version</p>
+      "! ABAP language version
+      "! $values {@link zif_aff_types_v1.data:co_abap_language_version_cloud}
+      "! $default {@link zif_aff_types_v1.data:co_abap_language_version_cloud.standard}
+      abap_language_version TYPE zif_aff_types_v1=>ty_abap_language_version,
+    END OF ty_header.
+
+  TYPES:
+    "! <p class="shorttext">Application Job Catalog Entry</p>
+    "! Name of the application job catalog entry
+    ty_catalog_name TYPE c LENGTH 40.
+
+  "! <p class="shorttext">Sign</p>
+  "! Include/exclude values (I/E)
+  "! $values {@link zif_aff_sajt_v1.data:co_sign}
+  TYPES ty_sign TYPE c LENGTH 1.
+
+  CONSTANTS:
+    "! <p class="shorttext">Sign</p>
+    "! Include/exclude values (I/E)
+    BEGIN OF co_sign,
+      "! <p class="shorttext">Exclude</p>
+      "! Exclude values
+      exclude TYPE ty_sign VALUE 'E',
+      "! <p class="shorttext">Include</p>
+      "! Include values
+      include TYPE ty_sign VALUE 'I',
+    END OF co_sign.
+
+  TYPES:
+    "! <p class="shorttext">Entry of Ranges Table</p>
+    "! Entry of ranges table
+    BEGIN OF ty_value_range_entry,
+      "! <p class="shorttext">Sign</p>
+      "! Include/exclude values (I/E)
+      sign      TYPE  ty_sign,
```

Comment:
Can't we make (at least sign) a reusable which can be used in SAJT and CHKV?

It would be even better to have a range as reusable, but this won't really work due to different types for `low` and `high`, will it?

Reply:
I checked with CHKV: Low and high are of type string there. If SAJT only allows 255 characters, we can't make one type for reuse

Reply:
As discussed, yesterday. Let's make ty_sign reusable. It refers also to an enumeration (for `include` and `exclude`) which would be reused then.

Reply:
I started to make `ty_sign` reusable (see next pull request in some minutes). When it comes out that SAJT can also use strings for `high` and `low`, I will do the same for `ty_value_range_entry`.

Reply:
```suggestion
      sign      TYPE  if_aff_types_v1=>ty_sign,
```
... and remove type `ty_sign` and constant `co_sign` in `if_aff_sajt_v1`